CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 + Output message to CLI when repo changes have been successfully published [#974](https://github.com/docker/notary/pull/974)
 + Improved error messages for client authentication errors and for the witness command [#972](https://github.com/docker/notary/pull/972)
 + Support for finding keys that are anywhere in the notary directory's "private" directory, not just under "private/root_keys" or "private/tuf_keys" [#981](https://github.com/docker/notary/pull/981)
++ Previously, on any error updating, the client would fall back on the cache.  Now we only do so if there is a network error or if the server is unavailable or missing the TUF data (and fails on invalid root rotation). [#982](https://github.com/docker/notary/pull/982)
```

Comment:
Can we split out the parenthesized bit for clarity?  Something like "Invalid TUF data will cause the update to fail, for example if there was an invalid root rotation`